neurovault/apps/statmaps/models.py

```diff
@@ -347,3 +351,33 @@ class Atlas(Image):
 
     class Meta:
         verbose_name_plural = "Atlases"
+
+
+class Similarity(models.Model):
```

Comment:
Not sure how Django handles associations, but it seems reasonable to keep this around as long as it's lazy loaded. If it's only used sporadically to look up the details of the similarity metric (e.g., once per results page). But it definitely shouldn't be eager-loaded every time a Comparison record is retrieved.